src/build/firefox.manifest.json

```diff
@@ -10,6 +10,7 @@
 	},
 	"permissions": [
 		"storage",
+		"*://localhost/*",
```

Comment:
might consider not adding this because changes in manifest trigger notifications to users